pytest_inmanta/plugin.py

```diff
@@ -289,17 +314,12 @@ def project_factory(
             raise
 
     with open(os.path.join(project_dir, "project.yml"), "w+") as fd:
-        metadata: typing.Mapping[str, object] = {
-            "name": "testcase",
-            "description": "Project for testcase",
-            "repo": repos,
-            "modulepath": modulepath,
-            "downloadpath": "libs",
-            "install_mode": install_mode.value,
-        }
-        yaml.dump(metadata, fd)
+        yaml.safe_dump(json.loads(project_metadata.json()), fd)
```

Comment:
The `pydantic.BaseModel.dict()` doesn't output something that can safely be dumped as yaml (`yaml.dump` would choke on enums) so I am serializing and de-serializing it here.  Any other suggestion?

Reply:
Nope, looks OK to me.